data/notifications.gschema.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <schemalist>
   <schema path="/io/elementary/notifications/" id="io.elementary.notifications">
+    <key type="as" name="applications">
```

Comment:
This is the same name as an existing subfolder which might be confusing. Not sure what a better name would be. Maybe "notifying-applications"?